jekyll/_cci2/docker-layer-caching.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: classic-docs
+title: "Docker Layer Caching"
+short-title: "Docker Layer Caching"
+order: 5
+---
+
+{% include beta-premium-feature.html%}
```

Comment:
Thanks, I intentionally dropped it to avoid repetition, because the whole doc is for this feature and just right above there will doc title. But if you think it still needs to be there, let me know, I'll add it.